doc/sphinx/language/core/conversion.rst

```diff
@@ -142,6 +212,29 @@ for :math:`x` an arbitrary variable name fresh in :math:`t`.
    because the type of the reduced term :math:`∀ x:\Type(2),~\Type(1)` would not be
    convertible to the type of the original term :math:`∀ x:\Type(1),~\Type(1)`.
 
+Examples
+~~~~~~~~
+
+   .. example:: Simple delta, fix, beta and match reductions
+
+      ``+`` represents ``Nat.add``:
```

Comment:
Maybe say "is a notation for" (with a link to the notations chapter) instead of "represents". And maybe add a sentence to note that it is defined as a fixpoint (`fix`).

Reply:
Think you mean ```:cmd:`Fixpoint` ```

Reply:
In practice, it is defined with the `Fixpoint` command. But what we see when doing the `Print` below is the resulting `fix` operator. That's why I was suggesting to mention that `Nat.add` is defined as a fixpoint (the actual command used doesn't matter). But only do it if you are comfortable writing about this.

Reply:
OK, actually you've already added something. LGTM.